Sources/Disposable.swift

```diff
@@ -182,15 +176,14 @@ public final class CompositeDisposable: Disposable {
 	///            `disposable` is `nil`.
 	@discardableResult
 	public func add(_ disposable: Disposable?) -> Disposable? {
-		guard let d = disposable, !d.isDisposed, !isDisposed else {
+		guard let _disposable = disposable, !_disposable.isDisposed, !isDisposed else {
```

Comment:
I think `d` is fine here; we don't need a longer name.

Reply:
I don't prefer to give meaningless variable names like `d`, but if the RAS naming conventions are are prefer `d`, I'll follow it.